### DISCUSSION TITLE
Cannot import repository

### DISCUSSION THREAD

**Discussion by thatoranzhevyy** [community/community#23363] [New to GitHub]
<p>I am trying to close an organization that I created in due time. I transfer the necessary repositories to my personal account. all other repositories were transferred without problems (I used the standard code import from github), but one project cannot be transferred in any way. gives an error message “There was an error pushing commits to GitHub.”.<br>
the repository weighs over 2 gigabytes. could this be a problem?</p>

**Comment by jsoref** [comment, marked as answer]
<p>How are you transferring the repository?</p>
<p><a href="https://docs.github.com/en/github/administering-a-repository/managing-repository-settings/transferring-a-repository#transferring-a-repository-owned-by-your-organization" class="onebox" target="_blank" rel="noopener">https://docs.github.com/en/github/administering-a-repository/managing-repository-settings/transferring-a-repository#transferring-a-repository-owned-by-your-organization</a></p>
<p>If you’re having trouble doing that, I’d suggest filing a ticket:</p>
<p><a href="https://support.github.com" class="onebox" target="_blank" rel="noopener">https://support.github.com</a></p>
